src/local.env

```diff
@@ -0,0 +1,18 @@
+KAFKA_STREAMS_APP_ID=pdp-kostas-kafka-streams
+KAFKA_STREAMS_APPLICATION_SERVER=localhost:8080
+KAFKA_STREAMS_STATE_DIR=/tmp/kafka-streams/pdp-kafkaStreams-8080
+KAFKA_BOOTSTRAP_SERVERS=localhost:9093
+KAFKA_REPLICATION_FACTOR=1
+KAFKA_NUM_STREAMS_THREADS=1
+KAFKA_NUM_STANDBY_REPLICAS=0
+KAFKA_COMMIT_INTERVAL_MS=100
+SCHEMA_REGISTRY_URL=http://localhost:8099
+KAFKA_STREAMS_COMPRESSION=lz4
+KAFKA_PRODUCER_MAX_REQUEST_SIZE=2196608
+PDP_LISTING_EVENTS_TOPIC=com.kostas.pdp.listings.events.avro
+PDP_PRICES_EVENTS_TOPIC=com.kostas.pdp.prices.events.avro
+
+SPRING_PROFILES_ACTIVE=intellij
```

Comment:
Any reason you used local.env files instead of spring profiles? (which you have as well)

Can achieve the same thing with multiple spring profiles (have a read up on them)

CMS uses local.envs for a different reason

Reply:
I thought I need the "local.env" files to define separate instances of the application. I guess the same can be done the couple of spring profiles. But yeah will have a read on it.